src/GameController/GCMicroGamepadSnapshot.cs

```diff
@@ -48,11 +82,25 @@ public partial class GCMicroGamepadSnapshot {
 		static extern bool GCMicroGamepadSnapShotDataV100FromNSData (out GCMicroGamepadSnapShotDataV100 snapshotData, /* NSData */ IntPtr data);
 
 		[Mac (10, 12)]
+		[Deprecated (PlatformName.iOS, message: "Use 'TryGetSnapshotData (NSData, out GCMicroGamepadSnapshotData)' instead.")]
+		[Deprecated (PlatformName.MacOSX, message: "Use 'TryGetSnapshotData (NSData, out GCMicroGamepadSnapshotData)' instead.")]
+		[Deprecated (PlatformName.TvOS, message: "Use 'TryGetSnapshotData (NSData, out GCMicroGamepadSnapshotData)' instead.")]
```

Comment:
^ this does not mention the version where the API are deprecated, which is important if you don't target the latest